earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -225,13 +225,25 @@ show_fusion_running_message()
 	echo -e "To use this installer to upgrade, you must stop all fusion services.\n"	
 }
 
+xml_file_get_xpath()
```

Comment:
The `common.sh` file is sourced by the various install scripts under `src/installer`.  I think that's a better place for the `xml_file_get_xpath()` function than copying it in each script.

However, I think we should do away with having code replicated between `installer` scripts, and RPM install scripts.  Each time you copy code from file to another, you tend to multiply the test steps you have to run by the number of times you've copied the code, and the same applies to every fix other modification to that code, as we have in this example.

You can install Fusion and Server using the same scripts that have been tested as part of RPM installations.  What you'd have to add is copying files and links that would be done by the package manager otherwise.